cli/cmd/root.go

```diff
@@ -199,6 +199,7 @@ type proxyConfigOptions struct {
 	enableExternalProfiles   bool
 	traceCollector           string
 	traceCollectorSvcAccount string
+	waitBeforeExitSeconds    uint
```

Comment:
Why not just make this `uint64`? Then you won't have to worry about type casting in `uintToString(v uint)` and `proxyWaitBeforeExitSeconds()`.

Reply:
First, explicit uint64 is quite big. Second, uint based on system architecture which changes the focus of where the decision is made. Third, doing anything here I'm trying to follow some, let's say, traditions that were made around. I'm not a Golang expert, so I learn from the project code. And around the variable there are many `uint`s, array of `uint`s and the only one `int64`. So I picked `uint`. KISS

Reply:
Thanks for the additional context. Since this is a CLI command (i.e. short-lived process), I am not too concerned about a 32 vs 64 bit local variable. Your `strconv.ParseUint(override, 10, 32)` is already returning a 64-bit uint anyway.

> KISS

I think removing all the type casting you have will help us to get closer to this goal.

Reply:
Oookay :)